bgpd/bgp_vty.c

```diff
@@ -4326,6 +4326,24 @@ DEFUN (neighbor_remote_as,
 	return peer_remote_as_vty(vty, argv[idx_peer]->arg,
 				  argv[idx_remote_as]->arg);
 }
+
+DEFPY (bgp_allow_martian,
+       bgp_allow_martian_cmd,
+       "[no]$no bgp allow-martian",
```

Comment:
Can we call this `nexthop-martian-allow` or `allow-martian-nexthop`?

(`nexthop-martian-allow` is reverse english but makes more sense when Tab'ing/`?`ing through the CLI)

Reply:
changed to allow-martian-nexthop